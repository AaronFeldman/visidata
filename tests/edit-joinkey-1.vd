sheet	col	row	longname	input	keystrokes	comment
	UsvSheet	delimiter	set-option	␞		
	UsvSheet	row_delimiter	set-option	␟		
			open-file	tests/data1.tsv	o	
			open-file	tests/data2.tsv	o	
data1			sheets-stack		S	
sheets		キdata2	open-row		^J	
data2			sheets-stack		S	
sheets		キdata2	select-row		s	
sheets		キdata1	select-row		s	
sheets			columns-selected		gC	
all_columns		キdata2,Key	select-row		s	
all_columns			key-selected		g!	
all_columns		キdata1,Key	select-row		s	
all_columns			key-selected		g!	
sheets			join-sheets	full	join-sheets	
data2+data1	A		key-col		!	
data2+data1			sort-keys-asc		g[	
data2+data1	A		key-col		!	
data2+data1	Key	キ2	edit-cell	4	e	
data2+data1			reload-sheet		^R	
cmdlog			save-sheet	tests/edit-joinkey-1.vd	^S	
