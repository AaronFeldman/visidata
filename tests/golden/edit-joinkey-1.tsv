Key	data2_C	data2_D	data1_A	data1_B
4	a2	b2	c1	d1
3	c2	d2		
1			a1	b1
2			e1	f1
